chart/files/pod-template-file.kubernetes-helm-yaml

```diff
@@ -54,7 +54,7 @@ spec:
         - mountPath: {{ include "airflow_dags_mount_path" . }}
           name: dags
           readOnly: true
-{{- if .Values.dags.persistence.enabled }}
+{{- if and .Values.dags.gitSync.enabled .Values.dags.persistence.enabled }}
```

Comment:
I am just not sure about one thing: do we actually need the check on `.Values.dags.persistence.enabled` ?

Reply:
True. We don't need that check. It's redundant anyway.